packages/icons/svgo-jsx-index.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs/promises");
```

Comment:
lets use ts file and run it like we run all bin scripts e.g. in SDK from /bin folder